golang/internal/mapper/grpc.go

```diff
@@ -284,7 +284,11 @@ func mapVolumes(in []*agent.Volume) []v1.Volume {
 		}
 
 		if in[i].Type != nil {
-			volume.Type = in[i].Type.String()
+			if *in[i].Type == common.VolumeType_MEM || *in[i].Type == common.VolumeType_TMP {
+				volume.Type = strings.ToLower(in[i].Type.String())
```

Comment:
Why do we need this?

Reply:
This was incorrect, **RWO**, **RWX** and **RW** are valid "real" storage types, while **tmp** and **memory** are synthetic ones. Around agents these are the values, unfortunately proto enum is always capitalized. This breaks no compatibility and works still.